src/EntryPoint/RestEntryPoints.ts

```diff
@@ -54,7 +54,9 @@ export class RestEntryPoints implements EntryPoints {
 
   public async entrypoints(
     projectId: string,
-    limit: number = 10
+    limit: number = 10,
+    connectivity?: string[],
+    status?: string[]
```

Comment:
Covert arguments to the object (especially if you gonna add other filters later), as the method allows a maximum of 4 args, ensuring readability

Reply:
will do, thanks!

Reply:
done!